cartographer_ros/cartographer_ros/map_builder_bridge.cc

```diff
@@ -209,7 +209,7 @@ visualization_msgs::MarkerArray MapBuilderBridge::GetTrajectoryNodesList() {
     marker.header.frame_id = node_options_.map_frame;
     marker.color.b = 1.0;
     marker.color.a = 1.0;
```

Comment:
Sounds good. In the past we had good experiences with just picking 8 visually distinct colors like this:
```
// Material Design primary colors.
constexpr uint32 kRed = 0xf44336;
constexpr uint32 kPurple = 0x9c27b0;
constexpr uint32 kIndigo = 0x3f51b5;
constexpr uint32 kTeal = 0x009688;
constexpr uint32 kGreen = 0x4caf50;
constexpr uint32 kAmber = 0xffc107;
constexpr uint32 kDeepOrange = 0xff5722;
constexpr uint32 kBrown = 0x795548;

// These colors are hand chosen to be visually distinct.
constexpr uint32 kColors[] = {kAmber,      kPurple, kRed,    kTeal,
                              kDeepOrange, kGreen,  kIndigo, kBrown};
```